modules/network/src/main/java/org/apache/ignite/internal/network/netty/ConnectionManager.java

```diff
@@ -117,6 +122,14 @@ public ConnectionManager(
      */
     public void start() throws IgniteInternalException {
         try {
+            boolean wasStarted = started.getAndSet(true);
```

Comment:
Have we discussed guarantees that start/stop should provide when called in parallel? Right now I can see a race